src/chia_dialect.rs

```diff
@@ -103,4 +107,12 @@ impl Dialect for ChiaDialect {
     fn apply_kw(&self) -> &[u8] {
         &[2]
     }
+
+    fn val_stack_limit(&self) -> usize {
```

Comment:
Is there a relationship between `val_stack_limit` and atom(pair/heap)_limit that could be expressed in code?

Reply:
I prefer to specify the actual limits, than trying to be clever and set a single limit that then translates to various limits

Reply:
Oh sure, I meant more like enforcing e.g. `assert heap_size > atom_limit * MIN_ATOM_SIZE` types of checks.